crates/core_simd/src/vector.rs

```diff
@@ -257,7 +265,8 @@ where
     /// let v = u32x4::from_slice(&source);
     /// assert_eq!(v.as_array(), &[1, 2, 3, 4]);
     /// ```
-    #[must_use]
```

Comment:
why delete `#[must_use]`? imho it's appropriate here...

Reply:
Wasn't intentional.